lib/v2/hbooker/chapter.js

```diff
@@ -0,0 +1,46 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const { id } = ctx.params;
+    const limit = Number.parseInt(ctx.query.limit) || 10;
+
+    const baseUrl = 'https://www.hbooker.com';
+
+    const { data: response } = await got(`${baseUrl}/book/${id}?arr_reverse=1`);
+    const $ = cheerio.load(response);
+
+    const list = $('div.book-chapter-list ul li a')
+        .slice(0, limit)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            return {
+                title: item.text(),
+                link: String(item.attr('href')),
```

Comment:
[`attr()`](https://cheerio.js.org/docs/api/classes/Cheerio#attr) will return string. 
No need to recast the same type.
docs: https://docs.rsshub.app/joinus/advanced/script-standard